src/DataFactory/DataFactoryV2/DataFactories/SetAzureDataFactoryCommand.cs

```diff
@@ -484,11 +505,23 @@ public override void ExecuteCmdlet()
             }
             FactoryIdentity factoryIdentity = new FactoryIdentity(factoryIdentityType, userAssignedIdentities: this.UserAssignedIdentity);
 
+            EncryptionConfiguration encryption = null;
+            if(!string.IsNullOrWhiteSpace(this.EncryptionVaultBaseUrl) && !string.IsNullOrWhiteSpace(this.EncryptionKeyName))
```

Comment:
As the description in design doc, if VaultBaseUrl is present, then keyName needs to be present as well. Any check for this?

Reply:
We are validating in backend and they'll get an error message. This is what we did for SDK as well to have a consistent experience.